src/screens/OtherProfile/index.js

```diff
@@ -164,13 +158,6 @@ const OtherProfile = () => {
   }, []);
 
   React.useEffect(() => {
-    netInfo.addEventListener((state) => {
-      if (state.isConnected) {
-        initData();
-      } else {
-        handleOfflineMode();
-      }
-    });
     initData();
   }, []);
 
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The removal of the `netInfo` event listener could lead to issues if the app needs to handle network changes. If the new hook `useOtherProfileScreenHooks` handles network status internally, then it's fine.



<!-- This is an auto-generated comment by OSS CodeRabbit -->